### PR TITLE
fix(whatsapp): content templates

### PR DESCRIPTION
**Pull request by cullenwatson**
* fetches their content templates via serverless
* allows user to select template from dropdown
* fixes conversation not opening if conversation already exists
image

### CODE REVIEW

**Review by dahamilton-twilio — COMMENTED** [review]
Thanks for you contribution @cullenwatson.

Happy to take a look next week and test it out. I am assuming your templates don't include variables? I would like to try and find the time to solve for that at some point but this is a great first step given the upcoming changes.

File: src/components/OutboundMessagePanel/OutboundMessagePanel.Container.js [hunk 1]
```diff
@@ -12,16 +12,13 @@ import {
...
 } from "@twilio-paste/core";
-
 import {
   Container,
   StyledSidePanel,
   DialerContainer,
   MessageContainer,
   SendMessageContainer,
   MessageTypeContainer,
-  OfflineContainer,
```
Comment (dahamilton-twilio) [inline src/components/OutboundMessagePanel/OutboundMessagePanel.Container.js]: Did the correct version get checked in or am I missing something? Aren't these required? thanks

Comment (asugai) [reply src/components/OutboundMessagePanel/OutboundMessagePanel.Container.js]: I just deployed this plugin using the updates that @cullenwatson made on this PR - those import do need to be added back in @dahamilton-twilio

Comment (dahamilton-twilio) [reply src/components/OutboundMessagePanel/OutboundMessagePanel.Container.js]: Thanks @asugai - appreciate the collaboration. It is on my todo list to get this into into main. Did you make any other changes for your use case that would be useful?

**Comment by cullenwatson** [comment]
Ah yea my templates didn't need variables. Prob needed to make this pr complete

**Comment by dahamilton-twilio** [comment]
> Ah yea my templates didn't need variables. Prob needed to make this pr complete

All good. Step in the right direction in advance of changes later in the year. Thanks

**Review by dahamilton-twilio — APPROVED** [review]

**Pull request merged**